scripts/url-check.R

```diff
@@ -80,14 +80,20 @@ get_urls <- function(file) {
   }
   url_list$knitr <- stringr::word(url_list$knitr, sep = "include_url\\(\"|\"\\)", 2)
   url_list$ottrpal <- stringr::word(url_list$ottrpal, sep = "include_slide\\(\"|\"\\)", 2)
-  url_list$markdown <- stringr::word(url_list$markdown, sep = "\\]\\(|\\)", 2)
+  url_list$markdown <- stringr::word(url_list$markdown, sep = "\\]\\(|\\)$", 2)
   url_list$markdown <- grep("http", url_list$markdown, value = TRUE)
 
   if (length(url_list$markdown_bracket) > 0 ){
     url_list$markdown_bracket <- paste0("http", stringr::word(url_list$markdown_bracket, sep = "\\]: http", 2))
   }
   url_list$other_http <- stringr::word(stringr::str_extract(url_list$other_http, url_pattern), sep = "\\]", 1)
-
+  
+  # Remove trailing comma if present (are there exceptions to this? Can a URL end in a comma?)
```

Comment:
`"(https://swirlstats.com),"` and `"https://swirlstats.com,"`

become

`"(https://swirlstats.com)"` and ` "https://swirlstats.com"`